packages/fx-core/src/plugins/resource/cicd/index.ts

```diff
@@ -69,7 +69,7 @@ export class CICDPluginV2 implements ResourcePlugin {
     tokenProvider: TokenProvider
   ): Promise<FxResult> {
     // add CI CD workflows for minimal app is not supported.
-    if (ctx.projectSetting && isPureExistingApp(ctx.projectSetting)) {
+    if (inputs.platform !== Platform.CLI_HELP && isPureExistingApp(ctx.projectSetting)) {
```

Comment:
What does this if condition mean?

Reply:
The original code is added by @qinezh to give a nice error in minimal app scenario. 
I found a bug (when CLI calls this getQuestions for CLI `--help`, it should not throw this error) when I implementing the CLI, so fix it along the way.
BTW, ctx.projectSetting is not an undefined type, so remove the useless if (ctx.projectSetting).